src/modules/NavBar/NavbarComponent.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { useUser } from "../user/UserContext";
+import ProtectedRoute from "../recipes/ProtectedRoute";
+
+export const Navbar = () => {
+  const { user, setUser } = useUser();
+
+  const handleLogout = () => {
+    setUser(null);
+  };
+
+  return (
+    <nav className="navbar">
+      <ul>
+        <li>
+          <NavLink to="/">Recipe Search</NavLink>
+        </li>
+        <li>
+          <NavLink to="/categories">Categories</NavLink>
+        </li>
+        {user ? (
+          <>
+            <li>
+              <ProtectedRoute>
```

Comment:
You don't need to use ProtectedRoute here, just add the link as normal. If the user is redirected to the protected route from navbar, ProtectedRoute component in the AppRouter will take care of it.